src/Tizen.NUI.Components/Theme/DefaultTheme.cs

```diff
@@ -128,6 +128,20 @@ public HashSet<ExternalThemeKeyList> GetExternalThemeKeyListSet()
                     .Add<Size>("/BottomLine/Size", (ViewStyle style, Size value) => ((TabButtonStyle)style).BottomLine.Size = value)
                     .Add<Position>("/BottomLine/Position", (ViewStyle style, Position value) => ((TabButtonStyle)style).BottomLine.Position = value)
                     .AddSelector<Color>("/BottomLine/BackgroundColor", (ViewStyle style, Selector<Color> value) => ((TabButtonStyle)style).BottomLine.BackgroundColor = value, ControlState.Selected),
+                
+                // TimePicker
+                (new ExternalThemeKeyList(typeof(TimePicker), typeof(TimePickerStyle)))
+                    .Add<Size>("/CellPadding", (ViewStyle style, Size value) => ((ViewStyle)style).Size = value)
```

Comment:
I am not sure if this is correct.
@rabbitfor Could you let us know if this is correct?

Reply:
I think this should be changed as follows.
```
((ViewStyle)style).Size = value)          ->            ((TimePicker)style).CellPadding = value)
```